src/scss/tools/_mixins.scss

```diff
@@ -16,7 +16,8 @@
   margin: 0;
   padding: 0;
   text-align: inherit;
-  font: inherit;
+  font-size: 1.125rem;
```

Comment:
same comment for font [ here](https://github.com/meranicosme/a-modern-eden/pull/2/commits/ba34206006f71912fca68db8e9fa03b07c5b8014#r323425278)

Reply:
Here for sure a font size system of some kind (variables, maps, etc.) should be considered and used.